pkg/telemetry/meta.go

```diff
@@ -68,3 +73,16 @@ func (m *Meta) SaveTo(fname string) error {
 
 	return ioutil.WriteFile(fname, data, 0644)
 }
+
+// GetTelemetryMeta read the telemeta from disk
+func GetTelemetryMeta(env *environment.Environment) (meta *Meta, fname string, err error) {
```

Comment:
Prefer to rename `GetTelemetryMeta` to `Meta` because the function is placed in the `telemetry` package.